crates/nu-command/src/filesystem/open.rs

```diff
@@ -103,7 +111,7 @@ impl Command for Open {
             let arg_span = path.span;
             // let path_no_whitespace = &path.item.trim_end_matches(|x| matches!(x, '\x09'..='\x0d'));
 
-            for path in nu_engine::glob_from(&path, &cwd, call_span, None)?.1 {
```

Comment:
Alternative number 3: using a `.map_err` here to convert a `DirectoryNotFound` to `FileNotFound`. It may be a bit hacky but minimally invasive.

Reply:
Seems like a good compromise to remove all the file_only bools and just use map_err here. Thanks for the idea!

Reply:
This should be done now